test/controllers/api/mentor_discussion_posts_controller_test.rb

```diff
@@ -21,30 +68,22 @@ class API::MentorDiscussionPostsControllerTest < API::BaseTestCase
     assert_equal expected, actual
   end
 
-  test "create should 403 unless user is mentor or student" do
+  test "create should 404 if the iteration does not exist" do
```

Comment:
Why has this test been deleted?

Reply:
I'll just copy the test for the index method. I decided to delete it because I put the 403 error now in the `use_mentor_discussion` before_action.